pkg/process/metadata/metadata_test.go

```diff
@@ -0,0 +1,40 @@
+package metadata
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/process/procutil"
+)
+
+func TestProcessMetadataProvider_ParseMetadata(t *testing.T) {
+	pmp := NewProcessMetadataProvider()
+
+	pmp.Extract(processData())
```

Comment:
Don't we need to add assert statements here or did you leave them out on purpose?